File: in_go/src/pattern_13.go
package src

import "fmt"

func Pattern_13() {
	row := 4
	column := 7
	symbol := 1

	for i := 0; i < row; i++ {
		symbol = 1 + i
		for j := 0; j < column; j++ {
			if j >= row-1-i && j <= row-1+i {
				fmt.Print(symbol)
				if j < row-1 {
					symbol++
				} else {
					symbol--
				}
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_rust/src/pattern_13.rs
pub fn pattern_13() {
    let row = 5;
    let column = 9;
    let mut symbol;

    for i in 0..row {
        symbol = 1 + i;
        for j in 0..column {
            if j >= row - 1 - i && j <= row - 1 + i {
                print!("{}", symbol);
                if j < row - 1 {
                    symbol += 1;
                } else {
                    symbol -= 1;
                }
            } else {
                print!(" ");
            }
        }
        println!();
    }
}
